src/common/productReleaseEntityAttributes.ts

```diff
@@ -38,4 +39,15 @@ export const productReleaseEntityAttributes = {
     id: 'productMilestone',
     title: 'Released In',
   },
+  commonPlatformEnumeration: {
+    id: 'commonPlatformEnumeration',
+    title: 'Common Platform Enumeration (CPE)',
+    tooltip:
+      "Common Platform Enumeration associated with the Product Release. CPEs are used to map packages that are security-relevant and delivered via security errata back to products and by the CVE Engine to map errata to containers when grading (e.g. 'cpe:/a:redhat:jboss_data_grid:7.3.5'). Can be retrieved from the Product Pages.",
+  },
+  productPagesCode: {
+    id: 'productPagesCode',
+    title: 'Product Pages Code',
+    tooltip: "Code associated with the Product Release in the Product Pages (e.g. 'rhdg-7-3.5').",
```

Comment:
Mind the preposition.
`tooltip: "Code associated with the Product Release on the Product Pages (e.g. 'rhdg-7-3.5').",`